tools/replay/replay.cc

```diff
@@ -25,6 +25,13 @@ Replay::Replay(QString route, QStringList allow, QStringList block, SubMaster *s
       s.push_back(it.name);
     }
   }
+
+  if (!allow_list.empty()) {
+    // the following events are needed for replay to work properly.
+    allow_list.insert(cereal::Event::Which::INIT_DATA);
```

Comment:
these added events will not pushed to stream(because they are not in the push socket_ array)
it's only used internally by replay.